### ISSUE TITLE
Package request: links

### ISSUE THREAD

**Issue by Symbian9** [termux/termux-packages#1834]
Please, add `links` for Termux, as alternative to `elinks` and `lynx`
*  http://links.twibright.com
  * http://links.twibright.com/download.php
  * http://links.twibright.com/download/ChangeLog

[![Packaging status](https://repology.org/badge/vertical-allrepos/links.svg)](https://repology.org/metapackage/links)

**Comment by Quasic** [comment]
There's also w3m, which can page web documents.

Was the last change in links a year ago?

**Comment by sdrausty** [comment]
It looks like links is currently available in Termux through Arch Linux PRoot.  This script is designed to install Arch Linux in just a few keystrokes. Tapping this link https://sdrausty.github.io/TermuxArch/setupTermuxArch.sh will transfer `setupTermuxArch.sh` to your device. Simply run `bash setupTermuxArch.sh` to install Arch Linux in Termux.  To install use `pacman -S links` and this will install links on device. 

![screenshot_20171129-235455](https://user-images.githubusercontent.com/27742457/33414804-f64801be-d560-11e7-97c8-0d5e4eb708f2.png)

**Comment by Symbian9** [comment]
> This script is designed to install Arch Linux in just a few keystrokes.

No, thanks.

I ask for `links` package for Termux, not for Arch Linux.

**Comment by KedalionDaimon** [comment]
This Arch Linux stuff is starting to look like **TOTAL SPAM**. How big is links - how big is Arch Linux? Why "Arch Linux", why not "GNURoot Debian"? WE ALL KNOW that "chroot is possible", but that is not the point of Termux, now, is it? I find OP's request a good idea... and the permanent suggestion of Arch Linux, which I have by now seen in a lot of other places, too, completely unsuitable.

**Comment by Quasic** [comment]
The goal is a termux package, but the proot methods are there in the interim, in case someone wants to resort to that while we figure out the termux package. More discussion on this concensus can be found in #183 and #1815. I know some of the arch stuff looks like ads, but this one appears more informative. No large colored text, at least. ;-)

**Comment by sdrausty** [comment]
@KedalionDaimon 

>WE ALL KNOW that "chroot is possible",  

Huh?  Don't assume what we all know and you will have a better day.  Smile.  It's contagious!

**Comment by Symbian9** [comment]
> The goal is a termux package

Exactly.

**Comment by sdrausty** [comment]
> but the proot methods are there in the interim, in case someone wants to resort to that while we figure out the termux package

Right on!  Use what works on smartphone and tablet in Termux today.

**Comment by Symbian9** [comment]
Any news on "links" package?

P.S.: *links 2.15* released few weeks ago

-

**Comment by Hax4us** [comment]
https://github.com/termux/termux-packages/pull/2317 Here is PR :)

**Comment by Quasic** [comment]
The installation procedure should start links with the calibration page, or is that just for graphics mode?

I'm not sure when I'll have time to try it, but I'm interested...

**Comment by Quasic** [comment]
Ooh, nice!

**Comment by Symbian9** [comment]
@Hax4us, awesome!

Has a question: is it possible to run it in graphics mode (`$ links -g http://example.com`) ?

**Comment by Auxilus** [comment]
yes it is!, I've compiled it with X11 support and it works fine! 

![img_20180408_195819](https://user-images.githubusercontent.com/23075789/38468686-4bee0d7c-3b67-11e8-8e32-b113303b591a.jpg)

**Comment by Hax4us** [comment]
@Symbian9 links supports graphics mode but in termux , we have no X11 or necessary libraries for graphics .. :(

**Comment by Hax4us** [comment]
But if you want graphics so u can build from source with x11 support ( as I know we have x11 libraries but unofficially provided by @xeffyr u can checkout his repository :)

**Comment by Symbian9** [comment]
> yes it is!, I've compiled it with X11 support and it works fine!

@Auxilus, I see VNC on your screenshot, so it is just remote desktop...

**Comment by Auxilus** [comment]
@Symbian9 that vnc server is running in termux

**Comment by Auxilus** [comment]
@Symbian9 if you are saying in spamming stuff, you better take a look at screenshot more carefully next time

**Comment by Symbian9** [comment]
Any news on `links2` packaging for Termux?

**Comment by Symbian9** [comment]
Links 2.16 released 
- http://links.twibright.com/download/links-2.16.tar.gz
- http://links.twibright.com/download/ChangeLog
```
=== RELEASE 2.16 ===

Sun Apr 29 17:12:24 CEST 2018 mikulas:

	Improve handling of the DELETE key
	Delete is used for both deleting and scrolling
	If the last keypress was captured in a form field, DELETE deletes
	If the last keypress was not captured DELETE scrolls

Sat Apr 28 21:49:08 CEST 2018 mikulas:

	Implement the bracketed paste mode - a text can be pasted only to input
	fields, the pasted text won't be misinterpreted as key commands

Sun Apr 22 17:28:05 CEST 2018 mikulas:

	Fix bugs found by coverity:
	* bad arguments to sizeof
	* missing int->unsigned long long conversions in image scaling
	* unchecked return of the close function
	* a possible NULL pointer dereference in directfb.c
	* a memory leak on invalid double number
	* missing va_end
	* non-working "No keepalive connection after POST request" option
	and some other discrepancies which didn't result in any violation

Sat Apr 21 23:13:49 CEST 2018 mikulas:

	Fix a crash in proxy authentication code (introduced in 2.15)

Thu Mar 29 20:06:50 CEST 2018 mikulas:

	Fixed internal error "invalid set_handlers call" on framebuffer
	if we suspend and terminate at the same time

Thu Mar 29 19:08:29 cet 2018 mikulas:

	When compiled without SVG, rewrite wikipedia svg math urls with png urls
```

**Comment by Symbian9** [comment]
@termux, any news on `links`/`links2` package for `termux`?

**Comment by Symbian9** [comment]
Links 2.17 released

- http://links.twibright.com/download/links-2.17.tar.gz
- http://links.twibright.com/download/ChangeLog

Please, make `links2` available for Termux!
